kbase/execution_engine2
# DATAUP-255 Added dev container

# Description of PR purpose/changes

* Update libs
* Create a docker-compose, config file, and deploy file, for local testing, removing dependency on mini_kb
* fixing travis build 
* refactor duplicate code in tests



# Testing Instructions
* Details for how to test the PR: 
* See if you can run tests via this setup and pass

# Dev Checklist:


# Updating Version and Release Notes (if applicable)
